src/index.ts

```diff
@@ -157,6 +158,11 @@ export interface AcrolinxEndpointProps {
    * @ignore
    */
   enableHttpLogging?: boolean;
+
+  /*
+   * Available in Acrolinx One
+   */
+  setTokenAsAuthHeader?: boolean;
```

Comment:
setTokenAsAuthHeader is sounding like a setter, if you can think of different naming.

Reply:
@pcdeshmukh Renamed to `useTokenAsAuthHeader`. Does it sound better?